cmd/ci-operator-prowgen/main.go

```diff
@@ -753,14 +751,14 @@ func generateJobsToDir(dir string, label jc.ProwgenLabel) func(configSpec *ciope
 		var orgConfig, repoConfig *config.Prowgen
 
 		if orgConfig, ok = cache[info.Org]; !ok {
-			if cache[info.Org], err = readProwgenConfig(filepath.Join(info.OrgPath, prowgenConfigFile)); err != nil {
+			if cache[info.Org], err = readProwgenConfig(filepath.Join(info.OrgPath, config.ProwgenConfigFile)); err != nil {
```

Comment:
This still stutters!!! You should not use the name of the package in the name of the var, ever.

```
config.ProwgenFile
```

Reply:
addressed